src/test/java/org/broadinstitute/hellbender/utils/downsampling/AlleleBiasedDownsamplingUtilsUnitTest.java

```diff
@@ -179,8 +179,8 @@ private static void testLoadFile(final File file, final Set<String> Samples, fin
     }
 
     @Test(dataProvider = "goodContaminationFiles")
-    public void testLoadContaminationFile(final Integer ArtificalBAMnumber, final Integer numberOfSamples) throws IOException {
-        final String ArtificialBAM = String.format("contamination.case.%d.txt", ArtificalBAMnumber);
+    public void testLoadContaminationFile(final int artificalBAMnumber, final int numberOfSamples) throws IOException {
```

Comment:
How did these changes make it into this branch?

Reply:
I was looking at the existing tests covering the contamination downsampling code, and noticed these minor issues (which resulting in errors/warnings in my IDE), so I fixed them.